<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TcpServerDemoClass</class>
 <widget class="QMainWindow" name="TcpServerDemoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TcpServerDemo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="lstMsg">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>621</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actStart"/>
   <addaction name="actStop"/>
   <addaction name="separator"/>
   <addaction name="actQuit"/>
  </widget>
  <action name="actStart">
   <property name="icon">
    <iconset resource="TcpServerDemo.qrc">
     <normaloff>:/TcpServerDemo/Resources/MYSL.ICO</normaloff>:/TcpServerDemo/Resources/MYSL.ICO</iconset>
   </property>
   <property name="text">
    <string>开启监听</string>
   </property>
  </action>
  <action name="actStop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="TcpServerDemo.qrc">
     <normaloff>:/TcpServerDemo/Resources/FOLDER.ICO</normaloff>:/TcpServerDemo/Resources/FOLDER.ICO</iconset>
   </property>
   <property name="text">
    <string>停止监听</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="icon">
    <iconset resource="TcpServerDemo.qrc">
     <normaloff>:/TcpServerDemo/Resources/OOFL.ICO</normaloff>:/TcpServerDemo/Resources/OOFL.ICO</iconset>
   </property>
   <property name="text">
    <string>退出程序</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TcpServerDemo.qrc"/>
 </resources>
 <connections/>
</ui>
